js/widget.js

```diff
@@ -38,6 +39,15 @@ const ModMap = {
   default: `sgv__last--`,
 };
 
+const updateAlwaysOnTopButton = (isAlwaysOnTop) => 
+  Buttons.alwaysOnTop.querySelector(`img`).src = isAlwaysOnTop ? `asset/icons8-pin-filled-25.png`:`asset/icons8-pin-hollow-25.png`;
```

Comment:
Assets should be set in HTML as a `src` attribute or in CSS as the `background-image` attribute.